.github/workflows/ci.yaml

```diff
@@ -41,6 +43,33 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+
+      - name: "Checkout corresponding ${{ matrix.homeserver }} branch"
+        if: ${{ matrix.homeserver == 'Synapse' }}
```

Comment:
We should perhaps put in a comment here to explain why Dendrite doesn't need this.